src/controllers/__tests__/sessionController.test.ts

```diff
@@ -142,21 +144,32 @@ describe("SessionController#listSessions", () => {
   beforeEach(() => {
     jest.clearAllMocks();
     mockSmembers.mockImplementationOnce((_, callback) => {
-      callback(null, [
-        `USER-${mockedUser.fiscal_code}-SESSION-${mockedUser.session_token}`
-      ]);
+      callback(null, [`SESSIONINFO-${mockedUser.session_token}`]);
     });
     mockMget.mockImplementationOnce((_, callback) => {
       callback(null, [JSON.stringify(expectedSessionInfo)]);
     });
   });
```

Comment:
usually we define mocks' default behaviour when we declare them, using `const myMock = jest.fn().mockImplementation( ... )`, then we just override this using `myMock.mockImplementationOnce( ... )` when the specific test case requires a different behaviour.

Reply:
fixed in f13d975